### PR TITLE
removing shares, text change on confirmation page, text change for co…

### PR DESCRIPTION
**Pull request by minayao**
…ming and finished projects

CA-332, 322, 306

### CODE REVIEW

**Comment by logophobia** [comment]
Which CA item is this?

File: CollAction/Views/Projects/Details.cshtml [hunk 1]
```diff
@@ -27,10 +27,20 @@
     <div class="container">
         @{
             string disableCommit = "";
+            string commitText = "Take Part";
             if (!Model.Project.IsActive)
             {
                 disableCommit = "disable-commit";
             }
+            if (Model.ExternalStatus == ProjectExternalStatus.ComingSoon)
```
Comment (logophobia) [inline CollAction/Views/Projects/Details.cshtml]: Maybe commit text should be a property of the viewmodel?

File: CollAction/Views/Projects/Details.cshtml [hunk 2]
```diff
@@ -27,10 +27,20 @@
...
+            {
+                commitText = "Coming soon";
+            }
+            else if (Model.ExternalStatus ==  ProjectExternalStatus.Closed)
+            {
+                commitText = "Project is finished";
+
```
Comment (logophobia) [inline CollAction/Views/Projects/Details.cshtml]: Empty line

File: CollAction/Views/Projects/Details.cshtml (hunk 1 above)
Comment (minayao) [reply CollAction/Views/Projects/Details.cshtml]: I left this one as is. The text is pretty specific to the view and the ExternalStatus might get used for something else in the future.

**Pull request merged**